aunt-leahs-app/src/styles.css

```diff
@@ -61,3 +61,23 @@ a {
     color: #36776f;
 }
 
+
+/**TEXTFIELD COMPONENT**/
+.textBoxLong {
+    width: 744px;
+    height: 54px;
+    margin: 12px 12px;
+}
+
+.textBoxShort {
+    width: 360px;
+    height: 54px;
+    margin: 12px 12px;
+}
```

Comment:
As per Cody's comments, it might be better for the TextInput to take a width parameter (there are a number of different text field widths in the original design).

Reply:
It seems like from the Figma design given that there will only be two widths for the text field: 360px and 744px. That's why I thought, in terms of ease of use, it might be easier to make it take in 'Short' or 'Long'.